src/styles/blocks/main.scss

```diff
@@ -0,0 +1,8 @@
+.main {
+  padding: 50px 40px;
+
+  display: grid;
+  grid-template-columns: repeat(auto-fill, 200px);
```

Comment:
Consider defining grid columns explicitly for different breakpoints to ensure responsive design. For example, use media queries to adjust the number of columns based on screen size.